plugins/calendar/src/main/java/de/unistuttgart/iaas/amyassist/amy/plugin/calendar/CalendarLogic.java

```diff
@@ -0,0 +1,243 @@
+			if (items.isEmpty()) {
+				return "No upcoming events found.";
+			}
+			for (Event event : items) {
+				DateTime start = event.getStart().getDateTime();
+				eventCalc(start, event, true);
+			}
+			return "You have following upcoming events:\n" + String.join("\n", this.eventList);
+		} catch (IOException e) {
+			this.logger.error("Sorry, I am not able to get your events.", e);
+			return "An error occured.";
+		}
+
+	}
+
+	/**
+	 * This method contains the logic to show the calendar events today and tomorrow
+	 * 
+	 * @param day
+	 * 			true if it is today, false for tomorrow	
+	 * @return the events of the chosen day
+	 */
+	public String getEventsByDay(boolean day) {
+		try {
+			this.eventList.clear();
+			DateTime now = new DateTime(System.currentTimeMillis());
+			Events events = this.calendarService.getService().events().list("primary").setTimeMin(now).setOrderBy("startTime")
+					.setSingleEvents(true).execute();
+			List<Event> items = events.getItems();
+			if (items.isEmpty()) {
+				return "No upcoming events found.";
+			}
+			if(!day) {
+				Date today = new Date();
+				LocalDateTime plusOne = today.toInstant().atZone(ZoneId.systemDefault()).toLocalDateTime();
+				plusOne = plusOne.plusDays(1);
+				now = new DateTime(Date.from(plusOne.atZone(ZoneId.systemDefault()).toInstant()));
+			}
+			for (Event event : items) {
+				String eventdatetime = "";
+				String eventdate = "";
+				if (event.getStart().getDateTime() != null) {
+					eventdatetime = event.getStart().getDateTime().toString().substring(0, 10);
+				} else if (event.getStart().getDate() != null) {
+					eventdate = event.getStart().getDate().toString().substring(0, 10);
+				}
+				if (eventdatetime.equals(now.toString().substring(0, 10))) {
+					DateTime start = event.getStart().getDateTime();
+					eventCalc(start, event, false);
+				} else if (eventdate.equals(now.toString().substring(0, 10))) {
+					eventCalc(null, event, false);
+				}
+			}
+			if(day) {
+				if (this.eventList.isEmpty()) {
+					return "There are no events today.";
+				}
+				return "You have following events today:\n" + String.join("\n", this.eventList);
+			}
+			if (this.eventList.isEmpty()) {
+				return "There are no events tomorrow.";
+			}
+			return "You have following events tomorrow:\n" + String.join("\n", this.eventList);
+		} catch (IOException e) {
+			this.logger.error("Sorry, I am not able to get your events.", e);
+			return "An error occured.";
+		}
+	}
+
+	/**
+	 * This method formats the date and time of the events and adds them into the list
+	 * 
+	 * @param start
+	 *            start time of the event
+	 * @param event
+	 *            data of the event
+	 * @param withDate
+	 *            determines if event output is with or without the date
+	 */
+	public void eventCalc(DateTime start, Event event, boolean withDate) {
+		String eventData;
+		String datePart = "";
+		if (start != null) {
+			String startData = start.toString();
```

Comment:
`DateTime` is in ISO-8601 / RFC 3339 format. so use `LocalDateTime.parse()` or other implementations of the specifications and don't write your own parser.